### PR TITLE
Remove the cert-manager dependency in favor of the service-ca-operator

### PR DESCRIPTION
**Pull request by timflannagan**
Updates the rukpak and PO YAML manifests and replaces the cert-manager dependency in favor of downstream's service-ca-operator for it's certificate needs.

Updates the rukpak manifests to the latest main branch tag.

### CODE REVIEW

File: config/rukpak/apis/webhooks/resources/deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: webhooks
           command: ["/webhooks"]
-          image: quay.io/operator-framework/rukpak:v0.8.0
+          image: quay.io/operator-framework/rukpak:main
```
Comment (tylerslaton) [inline config/rukpak/apis/webhooks/resources/deployment.yaml:20]: Are using main as a temporary measure? I mainly ask as this image may break in combination with the static manifests we're using here over time.

Comment (tylerslaton) [reply config/rukpak/apis/webhooks/resources/deployment.yaml:20]: Also, we may be able to turn a kustomize dial that allows us to set this image everywhere with one line making it easier to change overtime.

**Review by tylerslaton — APPROVED** [review]
This looks solid to me but I'm biased as we worked on it together. I'd encourage @exdx to take a look whenever he gets the chance.

/lgtm

File: config/rukpak/apis/webhooks/resources/deployment.yaml (hunk 1 above)
Comment (timflannagan) [reply config/rukpak/apis/webhooks/resources/deployment.yaml:20]: Yeah the better change here is using whatever kustomize can offer here. That would be a good follow-up change.

**Comment by timflannagan** [comment]
The e2e failures can be ignored. This repository hasn't been fully bootstrapped yet and we're waiting to be onboarded to ART's build management system before we can get e2e CI coverage.

**Comment by tylerslaton** [comment]
Tested against a 4.11 cluster and everything ran as expected.

/lgtm

File: config/rukpak/apis/webhooks/resources/deployment.yaml (hunk 1 above)
Comment (tylerslaton) [reply config/rukpak/apis/webhooks/resources/deployment.yaml:20]: https://github.com/openshift/platform-operators/issues/21

**Review by tylerslaton — APPROVED** [review]

**Comment by timflannagan** [comment]
We're still bootstrapping the repository before introducing PO to the payload, and this is one of the prerequisites to payload introduction. Manually adding the requisite no-ff labels.

/label px-approved
/label docs-approved
/label qe-approved

**Pull request merged**